docs/how-to-guides.md

```diff
@@ -32,6 +32,199 @@ Everything about test data generation and its adjustable parameters can
 be found in the [Generating a Test Dataset with niceML](
 generate-data.md#step-3-customizing-data-generation-optional)
 
+## How to write my own pipeline
+NiceML provides configurable, re-usable machine learning 
+pipelines. NiceML comes with pre-configured pipelines for object detection, 
+semantic semgmentation, regression, classification and others. But what if 
+your problem is not covered (yet) by the pipeline portfolio niceML offers?
+This guide will walk you through how to set up your first custom pipeline.
+The goal of the pipeline is to orchestrate a classifier that can distinguish
+between daisies and roses.
+
+1. Download the images from [Kaggle](https://www.kaggle.com/datasets/alxmamaev/flowers-recognition)
+and store the roses and daisies images in a `data` folder in your niceML project.
+2. Let us start configuring at the highest level of niceML: The job 
+configuration. For this, navigate to `configs/jobs/job_train/job_train_cls` 
+and create a new file `job_train_cls_binary_flowers.yaml`.
+3. Paste the content from `job_train_cls_binary.yaml` which is a good
```

Comment:
provide the whole path to the file or write something like '`job_train_cls_binary.yaml` which is in the same directory and is a good [...]'